### DISCUSSION TITLE
If I can use the cluster-api to manage my cluster work node?

### DISCUSSION THREAD

**Discussion by andyzheung** [kubernetes-sigs/cluster-api#7572] [Q&A] [answered]
I have a kubernetes cluster on vmware vsphere created by kubeadm. 
Now I want to manage the work node add and delete by cluster api.
If I can only mange work node by kubeadmconfig and use machinedeployment to manage it?
my k8s controller plane is not created by cluster api..?

**Comment by andyzheung** [comment]
I define VSphereCluster、KubeadmConfigTemplate、MachineDeployment、VSphereMachineTemplate crd,  can I use the exist k8s controller plane? and add node to the controller plane?

**Reply by sbueringer** [reply]
I don't think this is possible today.

cc @srm09

**Comment by andyzheung** [comment]
But I think this is a very common requirements.
because if I want to enhance my exist k8s cluster to get cluster autoscaler capibility, I just need to add or delete the work node to k8s controller plane..

**Comment by fabriziopandini** [comment, marked as answer]
discussion moved on https://github.com/kubernetes-sigs/cluster-api/issues/7573
